src/components/product-progress/handlers/extract-pnp-progress.handler.ts

```diff
@@ -95,16 +101,48 @@ export class ExtractPnpProgressHandler {
 
     // Update progress for report & product
     await Promise.all(
-      updates.map(async (input) => {
-        await this.progress.update(
-          {
-            ...input,
-            reportId: event.report.id,
-            // TODO this seems fine for now as only this variant will upload PnPs.
-            variant: Progress.FallbackVariant,
-          },
-          event.session,
-        );
+      updates.map(async ({ extracted, ...input }) => {
+        try {
+          await this.progress.update(
+            {
+              ...input,
+              reportId: event.report.id,
+              // TODO this seems fine for now as only this variant will upload PnPs.
+              variant: Progress.FallbackVariant,
+            },
+            event.session,
+          );
+        } catch (e) {
+          if (
+            !(
+              e instanceof AggregateError &&
+              e.message === 'Invalid Progress Input'
+            )
+          ) {
+            throw e;
+          }
+          for (const error of e.errors) {
+            if (!(error instanceof StepNotPlannedException)) {
+              continue;
+            }
```

Comment:
This does swallow this error
https://github.com/SeedCompany/cord-api-v3/blob/160c82ff9773d9de3033ee6b506fa7ba60ac85bd/src/components/product-progress/product-progress.service.ts#L165-L168

I don't think that `ProgressMeasurement.Number/Boolean` is ever used really.
So the 100% cap is already enforced by the PnP.
So I don't think this case is ever hit in practice.